### ISSUE TITLE
restrict fps

### ISSUE THREAD

**Issue by Bl4Cc4t** [nyaadevs/nyaa#372]
Since there already exist rules concerning the max. resolution a file should have, I’d like to have a rule for the fps as well. Seeing all this 60fps placebo stuff, I’d say that it isn’t just a bad meme anymore. It needs to stop.
Upscaling a DVD is bad enough. But increasing the fps *for no apparent reason* makes it even worse. Some people are obsessed with it though, I get that. 
ChibiGoku on [#963241](https://nyaa.si/view/963241) already constructively explained why it’s a dumb thing to do.
**TL;DR**: Please only allow anime using the original fps.

**Comment by CounterPillow** [comment]
>TL;DR: Please only allow anime using the original fps.

The wording of this would make it a problem for telecined material, where technically the awful telecined framerate is the *official* framerate. In those cases, we certainly do want to allow postprocessing like IVTC.

I do agree on the 60 fps stuff being a growing cancer though, but I'm going to guess there will be a lot of fighting over whether we want to police content more or allow dumb stuff because some people like dumb stuff and who are we to judge. Personally, I'm against the 60 fps interpolation crap because it leads to a situation where people start doing dumb shit to put buzzwords into their titles for attention, but there's a good case to be made for allowing it simply because they're not exactly harming anyone, and some people, as weird as it is, actually like this shit.

**Comment by Bl4Cc4t** [comment]
Hm, I agree with you, @CounterPillow. While not being an expert on encoding, I kinda get that there are some exceptions. And yes, there sure seem to be a lot of *fans* of this stuff. It’d certainly cause a lot of controversy. Welp, it seems it’s not as easy as I thought :V

**Comment by nyaazi** [comment]
Next time you will want to restrict what genres are uploaded.
It's pointless to do such restrictions on a public tracker.
